eng/pipelines/templates/jobs/ci.yml

```diff
@@ -198,7 +195,7 @@ jobs:
         inputs:
           filePath: "eng/scripts/CodeChecks.ps1"
           arguments: >
-           -ServiceDirectory ${{ parameters.ServiceToTest }}
+           -ServiceDirectory ${{ parameters.ServiceDirectory }}
```

Comment:
Won't this become "*"? I don't think CodeChecks or its subordinate scripts support that.

Reply:
For our new proposal, we don't support * anymore using $ServiceToTest.

Reply:
Okay. It does appear `join-path <path> '*' | resolve-path` resolves correctly.